lib/plugins/blocks.js

```diff
@@ -261,7 +261,7 @@ function inject (bot, { version, storageBuilder, hideErrors }) {
       bot.world.setColumn(packet.chunkX, packet.chunkZ, column)
     }
 
-    if (bot.supportFeature('dimensionDataIsAvailable')) {
+    if (bot.supportFeature('newLightingDataFormat')) {
```

Comment:
this is for 1.17 https://github.com/PrismarineJS/minecraft-data/blob/8a8c811c71790d75ec405429e5b50343cb77f88b/data/pc/common/features.json#L785

that contradicts the PR description

which is right?

Reply:
> that contradicts the PR description

Didn't say anything about this change in the pr description, but this change is correct: `loadParsedLight` is available starting from 1.17 (new lighting format)